test/gpu/native/reduction/combos.chpl

```diff
@@ -0,0 +1,54 @@
+const n = 10;
+
+config const useReduceEquals = false;
+config const useInitVal = false;
+
+on here.gpus[0] {
+  var A: [1..n] int;
+
+  foreach (i,a) in zip(A.domain, A) do a = i;
+
+  var expectedSumVar, expectedMinVar, expectedMaxVar: int;
+  for a in A do expectedSumVar +=  a;
+
+  var sumVar=0, minVar=max(int), maxVar=min(int): int;
+  if useInitVal {
+    sumVar = 100;
+    minVar = -100;
+    maxVar = 100;
```

Comment:
Overly fastidious suggestion: if you want to make the test more configurable you could set this to something like `n + 100` (just so if I updated the test to have an `n` to > 100 it would still work).